com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/HDGlobalSettingsWindow.cs

```diff
@@ -497,17 +454,15 @@ static void DrawMiscSettings(SerializedHDRenderPipelineGlobalSettings serialized
                 if (serialized.useDLSSCustomProjectId.boolValue)
                     EditorGUILayout.PropertyField(serialized.DLSSProjectId, Styles.DLSSProjectIdLabel);
 #endif
+                EditorGUI.BeginChangeCheck();
```

Comment:
@FrancescoC-unity I moved your code inside the indented scope to align with the rest of the UI - had to be done :)

I also cached the result of the method from reflection - please confirm you are okay with that change

Reply:
Sure